newt/builder/buildpackage.go

```diff
@@ -251,6 +252,16 @@ func (bpkg *BuildPackage) publicIncludeDirs(bspPkg *pkg.BspPackage) []string {
 
 		sdkIncls := bpkg.findSdkIncludes()
 		incls = append(incls, sdkIncls...)
+
+		settings := b.cfg.AllSettingsForLpkg(bpkg.rpkg.Lpkg)
+
+		inclDirs, err := bpkg.rpkg.Lpkg.PkgY.GetValStringSlice(
+			"pkg.include_dirs", settings)
+		util.OneTimeWarningError(err)
+
+		for _, src := range inclDirs {
```

Comment:
nitpick: `src` is an odd name to use here.  Something like `dir` feels more appropriate to me.

Reply:
Oops, I missed that. Thanks!